bb8/src/inner.rs

```diff
@@ -111,6 +114,10 @@ where
                 match self.inner.manager.is_valid(&mut conn).await {
                     Ok(()) => return Ok(conn),
                     Err(e) => {
+                        self.inner
+                            .statistics
+                            .connections_invalid_closed
+                            .fetch_add(1, Ordering::SeqCst);
```

Comment:
We can just make this another `StatsKind`, right?

Reply:
jaja, good point. Ill try to do that!